Combine CV update and template regeneration into a single step

somathias opened the pull request:
Also add a GHA to weekly pull ENA CVs.

Hammarn reviewed (dismissed):
If I understand correctly then `technical_metadata_fields_incl_ENA_CVs.json` is simply created by these scripts, correct?
I'm getting merge conflicts becasue it's been added, deleted and changed in different branches. It would be best to remove it from the checked git files if that is the case!?
Maybe it would be good to add it to `.gitignore`? Let me know what you think @somathias and I can get it done 🙂

somathias commented:
> If I understand correctly then `technical_metadata_fields_incl_ENA_CVs.json` is simply created by these scripts, correct? I'm getting merge conflicts becasue it's been added, deleted and changed in different branches. It would be best to remove it from the checked git files if that is the case!? Maybe it would be good to add it to `.gitignore`? Let me know what you think @somathias and I can get it done 🙂

Thanks for making me aware of this :) In the other PR I changed the file type for `technical_metadata_fields_incl_ENA_CVs` to yaml to clearly distinguish between input and output to the scripts. This confused git when doing the merge. I have now manually deleted the old `technical_metadata_fields_incl_ENA_CVs.json` to fix the conflict. If you approve @Hammarn, this should now be able to be merged.
